server/resource.go

```diff
@@ -48,17 +47,12 @@ func (s *Server) PostResource(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusBadRequest, err.Error())
 	}
 
-	dBytes, err := json.Marshal(data)
+	dBytes, err := json.Marshal(core.DataBlocks{data})
 	if err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, fmt.Errorf("failed to marshal: %w", err))
 	}
 
-	nc, err := client.New(s.bCtx)
-	if err != nil {
-		return echo.NewHTTPError(http.StatusInternalServerError, fmt.Errorf("failed to connect to the NATS server: %w", err))
```

Comment:
Should Swagger annotation be updated as well? I can't see `StatusInternalServerError` being returned anywhere else in this function, but maybe I'm just too tired too see it 😄